modules/mod_feed/services/provider.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @package     Joomla.Site
+ * @subpackage  mod_feed
+ *
+ * @copyright   (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+\defined('_JEXEC') or die;
```

Comment:
```suggestion
defined('_JEXEC') or die;
```
See [#31293. ](https://github.com/joomla/joomla-cms/pull/31293#issuecomment-720153712)

Reply:
Resolved, but build failed because of it I guess. How can I fix it?